dashboard/test/controllers/transfers_controller_test.rb

```diff
@@ -125,25 +125,6 @@ class TransfersControllerTest < ActionController::TestCase
     assert Follower.exists?(student_user: @word_student, section: @word_section)
   end
 
-  # TODO(asher): Determine what the desired behavior is, uncommenting this test
-  # or fixing the controller code appropriately.
-  # test "transferring a student to a section owned by a teacher the student "\
-  #      "already follows causes the student to transfer sections under that teacher" do
-  #   existing_section = create :section, user: @other_teacher, login_type: 'word'
-  #   Follower.create!(
-  #     user: @other_teacher,
-  #     student_user: @word_student,
-  #     section: existing_section
-  #   )
-  #   @params[:new_section_code] = @other_teacher_section.code
-  #   @params[:stay_enrolled_in_current_section] = false
-  #   post :create, params: @params
-  #
-  #   assert_response :no_content
-  #   refute Follower.exists?(student_user: @word_student, section: existing_section)
-  #   assert Follower.exists?(student_user: @word_student, section: @other_teacher_section)
-  # end
```

Comment:
This behavior is not desired [per @poorvasingal], so the (commented) test is being removed.